mlflow/utils/string_utils.py

```diff
@@ -12,3 +12,13 @@ def strip_suffix(original, suffix):
 
 def is_string_type(item):
     return isinstance(item, str)
+
+
+def truncate_str_from_middle(s, max_length):
+    assert max_length > 3
+    if len(s) <= max_length:
+        return s
+    else:
+        left_part_len = (max_length - 3) // 2
+        right_part_len = max_length - 3 - left_part_len
+        return f"{s[:left_part_len]}...{s[-right_part_len:]}"
```

Comment:
Does this work when `max_length = 4` and the string length is 5?

Reply:
It's a bit strange that the first character isn't shown. Can we require `max_length > 5` instead of `max_length > 3`?